http_service/tests/test_integration.py

```diff
@@ -28,7 +34,9 @@ def integration_test_single():
     response_json = response.json()
 
     if not response.ok:
-        raise Exception(f"Couldn't get an answer in {timeout} seconds: {response_json}")
+        raise TimeoutError(
+            f"Couldn't get an answer in {timeout} seconds: {response_json}"
+        )
```

Comment:
We check her for`not response.ok`, which include all possibilities and is not limited to timeout. Also, this is a test file, no need to create a custom exception class. We could use `requests.HTTPError` here:
```py
        raise requests.HTTPError(
            f"Couldn't get an answer in {timeout} seconds: {response_json}", response=response
        )
```

Reply:
Could you please add `response=response` as shown in the example?